lib/helpers/index.js

```diff
@@ -7,10 +7,20 @@ module.exports = {
      * Validate that response header exists
      * @param {object} request
      */
-    hasHeaders: function (request) {
+    hasHeaders: (request) => {
         return request.response !== null &&
             typeof request.response === 'object' &&
             typeof request.response.header === 'function' &&
-            true;
+            request;
+    },
+
+    /**
+     * Determine if default headers should be set
+     * @param {object} request
+     */
+    shouldSetDefaultCacheControl: (request = false) => {
+        return request && typeof request.response.headers['cache-control'] === 'undefined' &&
+            typeof request.route.settings.cache.expiresIn === 'undefined' &&
+            typeof request.route.settings.cache.expiresAt === 'undefined';
```

Comment:
How do we let people know that they can override the headers? Can you document in the readme?